dev-libs/libuv/libuv-1.9.1.recipe

```diff
@@ -1,16 +1,17 @@
 SUMMARY="A new platform layer for Node"
-DESCRIPTION="
-libuv is a multi-platform support library with a focus on asynchronous I/O. \
+DESCRIPTION="libuv is a multi-platform support library with a focus on asynchronous I/O. \
 It was primarily developed for use by Node.js, but it's also used by Luvit, \
 Julia, pyuv, and others."
-HOMEPAGE="http://github.com/libuv/libuv"
+HOMEPAGE="http://libuv.org/"
 COPYRIGHT="2009-2015 Ryan Dahl and others"
 LICENSE="MIT"
-REVISION="2"
-SOURCE_URI="git://github.com/hamishm/libuv#d48e9ae4bd504aadc25e26f06e83e9c2c86921ac"
-SOURCE_DIR="libuv-v1.2.0"
+REVISION="1"
+SOURCE_URI="https://github.com/libuv/libuv/archive/v$portVersion.tar.gz"
+SOURCE_DIR="libuv-$portVersion"
+CHECKSUM_SHA256="a6ca9f0648973d1463f46b495ce546ddcbe7cce2f04b32e802a15539e46c57ad"
+PATCHES="libuv-$portVersion.patchset"
 
-ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
 SECONDARY_ARCHITECTURES="x86"
```

Comment:
Since the recipe builds fine on x86_gcc2 **_primary_** arch, it _should_ also build on x86_gcc2 **_secondary_** arch, so it might be worth mentioning it in SECONDARY_ARCHITECTURES (if you see that it builds fine on x86_gcc2 secondary arch, of course):
```
SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```